src/App.js

```diff
@@ -1,12 +1,58 @@
-import React from 'react';
+import React, {useState, useEffect} from 'react';
+import axios from 'axios';
 import './App.css';
+import PokemonCard from "./components/PokemonCard"
 
 function App() {
-  return (
-    <div>
-      Begin hier met de opdracht!
-    </div>
-  );
+    const [pokemonData, setPokemonData] = useState([]);
+    const [urlPokemonDeck, setUrlPokemonDeck] = useState('https://pokeapi.co/api/v2/pokemon?offset=0&limit=20');
+
+    useEffect(() => {
+        async function fetchData() {
+            try {
+                const result = await axios.get(urlPokemonDeck);
+                setPokemonData(result.data);
+            } catch (error) {
+                console.error(error)
+            }
+        }
+
+        fetchData();
+    }, [urlPokemonDeck]);
+
+    return (
+        <>
+            {Object.keys(pokemonData).length > 0 &&
```

Comment:
Wat je hier doet is alle namen van de properties van een object teruggeven als een lijst waarmee je kijkt of het gevuld is of niet. Sowieso is een Object.keys op een array raar, omdat die geen keys heeft. Als je dus als default value `undefined` neemt, dan kun je hier gewoon `{pokemonData && <component />}` doen

Reply:
Ok.